cypress-shared/support/common/testcase.js

```diff
@@ -307,8 +307,16 @@ export function syncCheckoutUICustom() {
   it(`In ${prefix} - Sync Checkout UI Custom`, updateRetry(2), () => {
     cy.visit('admin/app/vtex-checkout-ui-custom/')
     cy.contains('Publish', { timeout: 25000 }).should('be.visible').click()
+    cy.getVtexItems().then((vtex) => {
+      cy.intercept('POST', `${vtex.baseUrl}/**`, (req) => {
+        if (req.body.operationName === 'GetHistory') {
+          req.continue()
```

Comment:
This will not work

**Reason** - 
Schedule Run got failed in this line -     
cy.contains(WORKSPACE, { timeout: 15000 }).should('be.visible')

But you added wait after that line.
So, if the same scenario happen again in future then it will still fail
+
You should wait for graphql that renders then workspace name in UI

Reference:
https://dashboard.cypress.io/projects/wk64au/runs/44/test-results/55d1c7cb-842e-4659-850d-a4bfead8625b